versioned_docs/version-2.0/resources/supportedChains.md

```diff
@@ -1,80 +1,218 @@
 # Supported Blockchains
 
+Don't see a blockchain you want?  Fill out this form for EVM chains and we'll add it: https://forms.gle/YQV5R7WoRyPk32xc7
```

Comment:
maybe we can copy these changes to `docs/resources/supportedChains.md` and add a section (or update a section title) to include `cayenne`. wdyt @Adarsh-Kumar28 ?

Reply:
Sure @sarahzdev I'll be raising another PR for addressing this as it involves updating the script